
 Network yanpitangui-dotnet-v2_backend  Creating
 Network yanpitangui-dotnet-v2_backend  Created
 Volume "yanpitangui-dotnet-v2_postgres-socket"  Creating
 Volume "yanpitangui-dotnet-v2_postgres-socket"  Created
 Volume "yanpitangui-dotnet-v2_uds-sockets"  Creating
 Volume "yanpitangui-dotnet-v2_uds-sockets"  Created
 Container yanpitangui-dotnet-v2-rinha-db-1  Creating
 Container yanpitangui-dotnet-v2-nats-1  Creating
 Container yanpitangui-dotnet-v2-nats-1  Created
 Container yanpitangui-dotnet-v2-rinha-db-1  Created
 Container backend-2  Creating
 Container backend-1  Creating
 Container yanpitangui-dotnet-v2-worker-1  Creating
 Container backend-1  Created
 Container backend-2  Created
 Container yanpitangui-dotnet-v2-nginx-1  Creating
 Container yanpitangui-dotnet-v2-worker-1  Created
 Container yanpitangui-dotnet-v2-nginx-1  Created
Attaching to backend-1, backend-2, nats-1, nginx-1, rinha-db-1, worker-1
nats-1  | [1] 2025/08/16 19:12:35.613329 [INF] Starting nats-server
nats-1  | [1] 2025/08/16 19:12:35.613377 [INF]   Version:  2.11.7
nats-1  | [1] 2025/08/16 19:12:35.613380 [INF]   Git:      [df44964]
nats-1  | [1] 2025/08/16 19:12:35.613382 [INF]   Cluster:  my_cluster
nats-1  | [1] 2025/08/16 19:12:35.613384 [INF]   Name:     NDBH2U7Q2ETIP2PTCYQMC7VJQUMUDQLNSS27IY5Z3MPJEBZH77LIS7PD
nats-1  | [1] 2025/08/16 19:12:35.613393 [INF]   ID:       NDBH2U7Q2ETIP2PTCYQMC7VJQUMUDQLNSS27IY5Z3MPJEBZH77LIS7PD
nats-1  | [1] 2025/08/16 19:12:35.613413 [INF] Using configuration file: /etc/nats/nats-server.conf (sha256:c7aaa34d64beb74e7c700fd3504b40b01deb1275f6567fb81ed3809ff03a5fe4)
nats-1  | [1] 2025/08/16 19:12:35.614134 [INF] Starting http monitor on 0.0.0.0:8222
nats-1  | [1] 2025/08/16 19:12:35.614316 [INF] Listening for client connections on 0.0.0.0:4222
nats-1  | [1] 2025/08/16 19:12:35.614502 [INF] Server is ready
nats-1  | [1] 2025/08/16 19:12:35.614661 [INF] Cluster name is my_cluster
nats-1  | [1] 2025/08/16 19:12:35.614695 [INF] Listening for route connections on 0.0.0.0:6222
rinha-db-1  | The files belonging to this database system will be owned by user "postgres".
rinha-db-1  | This user must also own the server process.
rinha-db-1  | 
rinha-db-1  | The database cluster will be initialized with locale "en_US.utf8".
rinha-db-1  | The default database encoding has accordingly been set to "UTF8".
rinha-db-1  | The default text search configuration will be set to "english".
rinha-db-1  | 
rinha-db-1  | Data page checksums are disabled.
rinha-db-1  | 
rinha-db-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
rinha-db-1  | creating subdirectories ... ok
rinha-db-1  | selecting dynamic shared memory implementation ... posix
rinha-db-1  | selecting default "max_connections" ... 100
rinha-db-1  | selecting default "shared_buffers" ... 128MB
rinha-db-1  | selecting default time zone ... Etc/UTC
rinha-db-1  | creating configuration files ... ok
rinha-db-1  | running bootstrap script ... ok
rinha-db-1  | performing post-bootstrap initialization ... ok
rinha-db-1  | initdb: warning: enabling "trust" authentication for local connections
rinha-db-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
rinha-db-1  | syncing data to disk ... ok
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | Success. You can now start the database server using:
rinha-db-1  | 
rinha-db-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
rinha-db-1  | 
rinha-db-1  | waiting for server to start....2025-08-16 19:12:46.783 UTC [55] LOG:  starting PostgreSQL 17.5 (Debian 17.5-1.pgdg130+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
rinha-db-1  | 2025-08-16 19:12:46.786 UTC [55] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1  | 2025-08-16 19:12:46.879 UTC [64] LOG:  database system was shut down at 2025-08-16 19:12:46 UTC
rinha-db-1  | 2025-08-16 19:12:47.080 UTC [55] LOG:  database system is ready to accept connections
rinha-db-1  |  done
rinha-db-1  | server started
rinha-db-1  | 2025-08-16 19:12:47.682 UTC [71] FATAL:  database "rinha" does not exist
backend-1   | backend-1
backend-2   | backend-2
rinha-db-1  | 2025-08-16 19:12:48.618 UTC [76] FATAL:  database "rinha" does not exist
nginx-1     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
rinha-db-1  | 2025-08-16 19:12:48.679 UTC [75] FATAL:  database "rinha" does not exist
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1     | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1     | /docker-entrypoint.sh: Configuration complete; ready for start up
rinha-db-1  | 2025-08-16 19:12:48.781 UTC [78] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.782 UTC [82] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.783 UTC [80] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.881 UTC [83] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.884 UTC [77] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.981 UTC [81] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:48.981 UTC [79] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.080 UTC [87] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.080 UTC [86] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.081 UTC [85] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.081 UTC [84] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.182 UTC [89] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.183 UTC [88] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.183 UTC [90] FATAL:  database "rinha" does not exist
worker-1    | PoolConfig { RouterPoolSize = 3 }
worker-1    | PipelineConfig { HandlePaymentParallelism = 20, PersistPaymentsParallelism = 10, GroupSize = 100, Timeout = 10 }
rinha-db-1  | 2025-08-16 19:12:49.380 UTC [91] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.382 UTC [92] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.383 UTC [93] FATAL:  database "rinha" does not exist
rinha-db-1  | 2025-08-16 19:12:49.384 UTC [94] FATAL:  database "rinha" does not exist
backend-2   | Unhandled exception. Npgsql.PostgresException (0x80004005): 3D000: database "rinha" does not exist
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__235.MoveNext() + 0xc43
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x50
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|214_1>d.MoveNext() + 0x9ab
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.Internal.NpgsqlConnector.<Open>d__214.MoveNext() + 0x873
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.PoolingDataSource.<OpenNewConnector>d__36.MoveNext() + 0x666
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.PoolingDataSource.<<Get>g__RentAsync|33_0>d.MoveNext() + 0xacf
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlConnection.<>c__DisplayClass165_0.<<StartBindingScope>g__StartBindingScopeAsync|0>d.MoveNext() + 0x35f
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlConnection.<<Open>g__PerformMultiplexingStartupCheck|42_1>d.MoveNext() + 0x3b6
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x120
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x2a8
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__9>d.MoveNext() + 0x57c
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<<Main>$>d__0.MoveNext() + 0x61e
backend-2   | --- End of stack trace from previous location ---
backend-2   |    at Program.<Main>(String[] args) + 0x26
backend-2   |   Exception data:
backend-2   |     Severity: FATAL
backend-2   |     SqlState: 3D000
backend-2   |     MessageText: database "rinha" does not exist
backend-2   |     File: postinit.c
backend-2   |     Line: 1032
backend-2   |     Routine: InitPostgres
backend-1   | Unhandled exception. Npgsql.PostgresException (0x80004005): 3D000: database "rinha" does not exist
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__235.MoveNext() + 0xc43
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x50
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|214_1>d.MoveNext() + 0x9ab
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.Internal.NpgsqlConnector.<Open>d__214.MoveNext() + 0x873
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.PoolingDataSource.<OpenNewConnector>d__36.MoveNext() + 0x666
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.PoolingDataSource.<<Get>g__RentAsync|33_0>d.MoveNext() + 0xacf
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlConnection.<>c__DisplayClass165_0.<<StartBindingScope>g__StartBindingScopeAsync|0>d.MoveNext() + 0x35f
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlConnection.<<Open>g__PerformMultiplexingStartupCheck|42_1>d.MoveNext() + 0x3b6
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x120
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Npgsql.NpgsqlDataSource.<OpenConnectionAsync>d__67.MoveNext() + 0x2a8
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__9>d.MoveNext() + 0x57c
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<<Main>$>d__0.MoveNext() + 0x61e
backend-1   | --- End of stack trace from previous location ---
backend-1   |    at Program.<Main>(String[] args) + 0x26
backend-1   |   Exception data:
backend-1   |     Severity: FATAL
backend-1   |     SqlState: 3D000
backend-1   |     MessageText: database "rinha" does not exist
backend-1   |     File: postinit.c
backend-1   |     Line: 1032
backend-1   |     Routine: InitPostgres
rinha-db-1  | CREATE DATABASE
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
nginx-1     | 2025/08/16 19:12:49 [crit] 30#30: *1 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:12:49 [crit] 30#30: *1 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
[Kbackend-2 exited with code 139
[Kbackend-1 exited with code 139
rinha-db-1  | CREATE TABLE
rinha-db-1  | CREATE INDEX
rinha-db-1  | 
rinha-db-1  | 
rinha-db-1  | 2025-08-16 19:12:50.180 UTC [55] LOG:  received fast shutdown request
rinha-db-1  | waiting for server to shut down....2025-08-16 19:12:50.184 UTC [55] LOG:  aborting any active transactions
rinha-db-1  | 2025-08-16 19:12:50.186 UTC [55] LOG:  background worker "logical replication launcher" (PID 67) exited with exit code 1
rinha-db-1  | 2025-08-16 19:12:50.280 UTC [62] LOG:  shutting down
rinha-db-1  | 2025-08-16 19:12:50.283 UTC [62] LOG:  checkpoint starting: shutdown immediate
rinha-db-1  | 2025-08-16 19:12:50.509 UTC [62] LOG:  checkpoint complete: wrote 927 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.192 s, sync=0.024 s, total=0.229 s; sync files=311, longest=0.004 s, average=0.001 s; distance=4277 kB, estimate=4277 kB; lsn=0/191AF38, redo lsn=0/191AF38
rinha-db-1  | 2025-08-16 19:12:50.580 UTC [55] LOG:  database system is shut down
rinha-db-1  |  done
rinha-db-1  | server stopped
rinha-db-1  | 
rinha-db-1  | PostgreSQL init process complete; ready for start up.
rinha-db-1  | 
rinha-db-1  | 2025-08-16 19:12:50.784 UTC [1] LOG:  starting PostgreSQL 17.5 (Debian 17.5-1.pgdg130+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
rinha-db-1  | 2025-08-16 19:12:50.785 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
rinha-db-1  | 2025-08-16 19:12:50.785 UTC [1] LOG:  listening on IPv6 address "::", port 5432
rinha-db-1  | 2025-08-16 19:12:50.792 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
rinha-db-1  | 2025-08-16 19:12:50.801 UTC [101] LOG:  database system was shut down at 2025-08-16 19:12:50 UTC
rinha-db-1  | 2025-08-16 19:12:50.886 UTC [1] LOG:  database system is ready to accept connections
nginx-1     | 2025/08/16 19:12:54 [error] 30#30: *4 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:12:59 [error] 30#30: *5 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:04 [crit] 30#30: *6 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:04 [crit] 30#30: *6 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:09 [error] 30#30: *9 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:14 [error] 30#30: *10 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:19 [crit] 30#30: *11 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:19 [crit] 30#30: *11 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:24 [error] 30#30: *14 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:29 [error] 30#30: *15 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:34 [crit] 30#30: *16 connect() to unix:/sockets/backend-2.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-2.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:34 [crit] 30#30: *16 connect() to unix:/sockets/backend-1.sock failed (2: No such file or directory) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://unix:/sockets/backend-1.sock:/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:39 [error] 30#30: *19 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
nginx-1     | 2025/08/16 19:13:44 [error] 30#30: *20 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://api/payments-summary", host: "localhost:9999"
[Knginx-1 exited with code 0
[Kworker-1 exited with code 0
nats-1      | [1] 2025/08/16 19:13:50.399255 [INF] Trapped "terminated" signal
nats-1      | [1] 2025/08/16 19:13:50.399427 [INF] Initiating Shutdown...
nats-1      | [1] 2025/08/16 19:13:50.399589 [INF] Server Exiting..
rinha-db-1  | 2025-08-16 19:13:50.401 UTC [1] LOG:  received fast shutdown request
rinha-db-1  | 2025-08-16 19:13:50.404 UTC [1] LOG:  aborting any active transactions
rinha-db-1  | 2025-08-16 19:13:50.479 UTC [1] LOG:  background worker "logical replication launcher" (PID 104) exited with exit code 1
rinha-db-1  | 2025-08-16 19:13:50.481 UTC [99] LOG:  shutting down
rinha-db-1  | 2025-08-16 19:13:50.484 UTC [99] LOG:  checkpoint starting: shutdown immediate
rinha-db-1  | 2025-08-16 19:13:50.510 UTC [99] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.007 s, sync=0.005 s, total=0.030 s; sync files=5, longest=0.003 s, average=0.001 s; distance=8 kB, estimate=8 kB; lsn=0/191CF80, redo lsn=0/191CF80
[Knats-1 exited with code 0
rinha-db-1  | 2025-08-16 19:13:50.580 UTC [1] LOG:  database system is shut down
